vaadin-ordered-layout-flow-parent/vaadin-ordered-layout-flow/src/test/java/com/vaadin/flow/component/orderedlayout/tests/ScrollerTest.java

```diff
@@ -84,4 +86,14 @@ public void setNullScrollDirection_NullPointerExceptionIsThrown() {
         scroller.setScrollDirection(null);
     }
 
+    @Test
+    public void setEnabled_disableChildren() {
```

Comment:
It isn't immediately clear why `Focusable` interface is tested by using `setEnabled` API.
Also, using external components such as `TextField` in unit tests should be avoided.

Consider using the following test as an example, I think it should be enough:

https://github.com/vaadin/flow-components/blob/4a324e3a537f9807a7af5a4dd170b3f4535c6556/vaadin-combo-box-flow-parent/vaadin-combo-box-flow/src/test/java/com/vaadin/flow/component/combobox/ComboBoxBaseTest.java#L54-L57

Reply:
I tested the setEnabled just because the behavior related with it, i.e. disable its children, is not trivial.

About the user of external components, TextField was just a random component I remembered, I've just replaced it with 'Input', which is already used in the project. If this test is not considered relevant, I might just remove it.